backend/cmd/server/bootstrap/flows/authentication/auth_flow_sms_with_username.json

```diff
@@ -1,53 +0,0 @@
-{
-    "name": "SMS OTP Authentication with Username Flow",
```

Comment:
Removed this graph as it is redundant. We don't need to created this during bootstrap scripts